docs/pt/docs/how-to/conditional-openapi.md

```diff
@@ -0,0 +1,58 @@
+# OpenAPI condicional
+
+Se necessário, você pode usar configurações e variáveis ​​de ambiente para configurar o OpenAPI condicionalmente, dependendo do ambiente, e até mesmo desativá-lo completamente.
+
+## Sobre segurança, APIs e documentos
+
+Ocultar suas interfaces de usuário de documentação na produção *não deveria* ser a maneira de proteger sua API.
+
+Isso não adiciona nenhuma segurança extra à sua API; as *operações de rotas* ainda estarão disponíveis onde estão.
+
+Se houver uma falha de segurança no seu código, ela ainda existirá.
+
+Ocultar a documentação apenas torna mais difícil entender como interagir com sua API e pode dificultar sua depuração na produção. Pode ser considerado simplesmente uma forma de <a href="https://en.wikipedia.org/wiki/Security_through_obscurity" class="external-link" target="_blank">Segurança através da obscuridade</a>.
+
+Se você quiser proteger sua API, há várias coisas melhores que você pode fazer, por exemplo:
+
+* Certifique-se de ter modelos Pydantic bem definidos para seus corpos de solicitação e respostas.
+* Configure quaisquer permissões e funções necessárias usando dependências.
+* Nunca armazene senhas em texto simples, apenas hashes de senha.
+* Implemente e use ferramentas criptográficas bem conhecidas, como tokens Passlib e JWT, etc.
```

Comment:
it's JWT tokens, not Passlib tokens.

The translation would be:

"... como tokens JWT e Passlib"